ambient-poly.py

```diff
@@ -102,12 +118,35 @@ def longPoll(self):
                 self.set_driver(node, 'GV3', d, 'monthlyrainin')
                 self.set_driver(node, 'GV4', d, 'yearlyrainin')
             elif self.nodes[node].id == 'light':
-                self.set_driver(node, 'ST', d, 'uv')
+                self.set_driver(node, 'ST', d, 'uv')            
```

Comment:
This looks like the only change was extra white space added to the end of the line.